packages/dx-react-scheduler/docs/guides/fundamentals.md

```diff
@@ -0,0 +1,43 @@
+# React Scheduler - Fundamentals
+
+The [DevExtreme React Scheduler](https://devexpress.github.io/devextreme-reactive/react/scheduler) component that displays appointments and provide a functionality to manage data.
+
+## Overview
+
+The React Scheduler has built-in functionality for appointment layouting in different views. The scheduler contains easily customizable UI elements such as date navigator, view switcher, all-day panel, appointment tooltip, and appointment form. The [editing](../reference/editing-state.md) functionality is supported, as well. Moreover, you can specify which one of [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) operations is available.
+
+The following image describes the main UI elements of the React Scheduler.
+
+<p align="center">
+  <img class="img-responsive" src="../../img/scheduler-elements.png">
+</p>
+
+The React Scheduler adheres to plugin-based architecture. This says that every feature is required a special plugin. See also the Plugin Overview guide. Next plugins implement most common features.
```

Comment:
> every feature is required

every feature requires

I'd say something like: 'every feature is encapsulated in a particular plugin'